flows/constants.py

```diff
@@ -27,7 +27,7 @@
 KLIMA_NBO_ADDRESS = "0x251ca6a70cbd93ccd7039b6b708d4cb9683c266c"
 KLIMA_MCO2_ADDRESS = "0x64a3b8ca5a7e406a78e660ae10c7563d9153a739"
 KLIMA_BCT_ADDRESS = "0x9803c7ae526049210a1725f7487af26fe2c24614"
-BCT_USDC_ADDRESS = "0x1e67124681b402064cd0abe8ed1b5c79d2e02f64"
+BCT_USDC_ADDRESS = "0x9803c7ae526049210a1725f7487af26fe2c24614"
```

Comment:
Technically this is the BCT_KLIMA pool, we just provide the USD price on the subgraph natively...

Reply:
Done in d4c8d7f